File: Medium/JavaScript/80. Remove Duplicates from Sorted Array II.js
var removeDuplicates = function(nums) {
    if(nums.length < 3)
        return nums.length;
    
    let i = 2;
    for(let j = i; j < nums.length; j++){
        if(nums[i - 2] != nums[j]){
            nums[i] = nums[j];
            i++;
        }
    }

    return i;
};

File: Medium/Python/80. Remove Duplicates from Sorted Array II.py
class Solution(object):
    def removeDuplicates(self, nums):
        if len(nums) < 3:
            return len(nums)
        
        i = 2
        for j in range(i, len(nums)):
            if nums[i - 2] != nums[j]:
                nums[i] = nums[j]
                i += 1
        
        return i
